ext/ext-api/api-qldb/src/main/resources/META-INF/dirigible/qldb/extensions/modules.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022 SAP SE or an SAP affiliate company and Eclipse Dirigible contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Eclipse Dirigible contributors
+ * SPDX-License-Identifier: EPL-2.0
+ */
+let registry = require('platform/v4/registry');
+exports.getContent = function () {
+    let file = registry.getText("qldb/extensions/modules.json");
```

Comment:
Use `const` and rename the variable to something more meaningful, it's not a file